lib/companies_web/views/job_view.ex

```diff
@@ -17,4 +19,18 @@ defmodule CompaniesWeb.JobView do
       "input"
     end
   end
+
+  def remote_ribbon(%{remote: true}), do: "has-ribbon"
+  def remote_ribbon(_), do: ""
+
+  def link_title(url) do
+    url
+    |> String.replace("http://", "")
+    |> String.replace("https://", "")
+    |> String.replace("www.", "")
+  end
+
+  def url_with_scheme("http://" <> _ = url), do: url
```

Comment:
Do we want to focus https for all urls?

Reply:
We are not, we just make it the default when the user is not giving us any. (I don't remember why we originally did that for the companies though)